pages/common/st-util.md

```diff
@@ -0,0 +1,15 @@
+# st-util
+
+> Run GDB server to interact with STM32 ARM Cortex microcontoller.
+
+- Run GDB server on port 4500:
+
+`st-util -p 4500`
```

Comment:
According to our conventions we use spelled out versions of options when available. Lets use `--listen_port` instead of `-p`. Also wrap 4500 within {{}}